npm ERR! peer dep missing: @babel/preset-env@^7.1.6, required by jscodeshift@0.9.0

### Version
4.4.5






### Environment info
```
  System:
    OS: macOS Mojave 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  Binaries:
    Node: 14.0.0 - ~/.nvm/versions/node/v14.0.0/bin/node
    Yarn: Not Found
    npm: 6.14.4 - ~/.nvm/versions/node/v14.0.0/bin/npm
  Browsers:
    Chrome: 83.0.4103.106
    Edge: Not Found
    Firefox: Not Found
    Safari: 13.1.1
  npmGlobalPackages:
    @vue/cli: 4.4.5
```


### Steps to reproduce
$ npm list -g --depth 0  
/Users/brand/.nvm/versions/node/v14.0.0/lib
├── @vue/cli@4.4.5
└── npm@6.14.4

npm ERR! peer dep missing: @babel/preset-env@^7.1.6, required by jscodeshift@0.9.0

### What is expected?
No error for peer deps

### What is actually happening?
npm ERR! peer dep missing: @babel/preset-env@^7.1.6, required by jscodeshift@0.9.0

<!-- generated by vue-issues. DO NOT REMOVE -->

--- a/packages/@vue/cli/package.json
+++ b/packages/@vue/cli/package.json
@@ -24,6 +24,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@babel/preset-env": "^7.9.6",
     "@vue/cli-shared-utils": "^4.4.5",
     "@vue/cli-ui": "^4.4.5",
     "@vue/cli-ui-addon-webpack": "^4.4.5",
@@ -44,7 +45,7 @@
     "isbinaryfile": "^4.0.6",
     "javascript-stringify": "^1.6.0",
     "js-yaml": "^3.13.1",
-    "jscodeshift": "^0.9.0",
+    "jscodeshift": "^0.10.0",
     "leven": "^3.1.0",
     "lodash.clonedeep": "^4.5.0",
     "lru-cache": "^5.1.1",

